### ISSUE TITLE
libbi 1.4.4 fails to run in Homebrew

### ISSUE THREAD

**Issue by fxcoudert** [lawmurray/LibBi#45]
I am a maintainer for Homebrew, a macOS distro that is shipping libbi. We are seeing our tests of this software fail, like this:

```
$ cat Test.bi                                                                                                                 
model Test {
  param theta, sigma2;
  noise w;
  state x;

  sub parameter {
    theta ~ gaussian();
    sigma2 ~ inverse_gamma();
  }

  sub initial {
    x ~ gaussian();
  }

  sub transition {
    w ~ gaussian(0.0, sqrt(sigma2));
    x <- theta*x + w;
  }
}
$ cat test.conf                                                                                                               
--target prior
--model-file Test.bi
--end-time 1
--nsamples 32
--output-file test.nc
$ /usr/local/Cellar/libbi/1.4.4_2/bin/libbi sample @test.conf                                                                 
Error: make failed with return code 2, see /private/tmp/libbi-test-20190224-16666-zvp6u5/.Test/build_assert_openmp_sm_30/make.log for details
```

The `make.log` file is here: https://gist.github.com/fxcoudert/c0255e3c521c651e9d1579eb8453ae0b

It has many errors, including:

```
/usr/local/include/thrust/system/omp/detail/for_each.inl:53:3: error: implicit instantiation of undefined template 'thrust::detail::STATIC_ASSERTION_FAILURE<false>'
  THRUST_STATIC_ASSERT( (thrust::detail::depend_on_instantiation<RandomAccessIterator,
```

**Comment by fxcoudert** [comment]
`libbi sample @test.conf --disable-openmp` does pass, on the other hand.

**Comment by sbfnk** [comment]
This should be fixed by aadb65c9e756723782e4a8f1a104d74e2086a0c6. See also https://github.com/Homebrew/homebrew-core/pull/41715.
